README.md

```diff
@@ -1,355 +1,291 @@
-def trainer(
-    X_train: np.ndarray,
-    y_train: np.ndarray,
-) -> ClassifierMixin:
-    """Train a simple sklearn classifier for the digits dataset."""
-    model = get_digits_model()
-    model.fit(X_train, y_train)
-    return model
-
-
-@step
-def evaluator(
-    X_test: np.ndarray,
-    y_test: np.ndarray,
-    model: ClassifierMixin,
-) -> float:
-    """Calculate the accuracy on the test set"""
-    test_acc = model.score(X_test, y_test)
-    print(f"Test accuracy: {test_acc}")
-    return test_acc
-
-
-@pipeline
-def mnist_pipeline(
-    importer,
-    trainer,
-    evaluator,
-):
-    """Links all the steps together in a pipeline"""
-    X_train, X_test, y_train, y_test = importer()
-    model = trainer(X_train=X_train, y_train=y_train)
-    evaluator(X_test=X_test, y_test=y_test, model=model)
-
-
-pipeline = mnist_pipeline(
-    importer=importer(),
-    trainer=trainer(),
-    evaluator=evaluator(),
-)
-pipeline.run()
-```
++ Train, evaluate, deploy, and embed a model in an inference pipeline.
++ Automatically track and version data, models, and other artifacts.
++ Track model hyperparameters and metrics with experiment tracking tools.
++ Measure and visualize train-test skew, training-serving skew, and data drift.
 
-# :racehorse: Get a guided tour with `zenml go`
+## 🍰 ZenBytes
+New to MLOps? Get up to speed by visiting the [ZenBytes](https://github.com/zenml-io/zenbytes) repo.
 
-For a slightly more in-depth introduction to ZenML, taught through Jupyter
-notebooks, install `zenml` via pip as described above and type:
+>ZenBytes is a series of short practical MLOps lessons taught using ZenML. 
+>It covers many of the [core concepts](https://docs.zenml.io/getting-started/core-concepts) widely used in ZenML and MLOps in general.
 
-```shell
-zenml go
-```
+## 📜 ZenFiles
+Already comfortable with ZenML and wish to elevate your pipeline into production mode? Check out [ZenFiles](https://github.com/zenml-io/zenfiles).
 
-This will spin up a Jupyter notebook that showcases the above example plus more
-on how to use and extend ZenML.
+>ZenFiles is a collection of production-grade ML use-cases powered by ZenML. They are fully fleshed out, end-to-end projects that showcase ZenML's capabilities. They can also serve as a template from which to start similar projects.
 
 # 👭 Collaborate with your team
 
-ZenML is built to support teams working together. The underlying infrastructure
-on which your ML workflows run can be shared, as can the data, assets and
-artifacts that you need to enable your work. ZenML Profiles offer an easy way to
-manage and switch between your stacks. The ZenML Server handles all the
-interaction and sharing and you can host it wherever you'd like.
+ZenML is built to support teams working together. 
+The underlying infrastructure on which your ML workflows run can be shared, as can the data, assets, and artifacts in your workflow. 
+
+You can quickly share your stack with anyone with ZenML by exporting them with:
```

Comment:
The concept of "stack" has not been introduced yet, maybe you can add a short intro.

```suggestion
In ZenML, a Stack represents a set of configurations for your MLOps tools and infrastructure. You can quickly share your ZenML stack with anyone by exporting the stack:
```